docs/pages/button.js

```diff
@@ -259,245 +229,481 @@ card(
-          id="disable-buttons"
-          switched={disabled}
-        />
-        <Box paddingX={2} flex="grow">
-          <Label htmlFor="disable-buttons">
-            <Text>Disable buttons</Text>
-          </Label>
-        </Box>
-      </Flex>
-      <Flex gap={2}>
-        <Switch
-          onChange={() => setTabIndex(!tabIndex)}
-          id="unreachable-buttons"
-          switched={tabIndex}
-        />
-        <Box paddingX={2} flex="grow">
-          <Label htmlFor="unreachable-buttons">
-            <Text>Remove from keyboard navigation with tabIndex</Text>
-          </Label>
-        </Box>
-      </Flex>
-    </Flex>
-  );
-}
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Show the full text on Buttons. Buttons should be stacked when they cannot be displayed side by side."
+        defaultCode={`
+<Flex gap={2} direction="column" alignContent="stretch">
+  <Button text="Create account" size="lg" color="red" fullWidth />
+  <Button text="View settings" size="lg" color="gray" fullWidth />
+</Flex>
 `}
-  />,
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Truncate the Button text. In rare instances where Buttons must remain on one line, truncate the text on the secondary Button before truncating on the primary Button."
+        defaultCode={`
+<Flex gap={2}>
+  <Button text="Kontoeinst..." size="lg" color="gray" />
+  <Button text="Neues We..." size="lg" color="red" />
+</Flex>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Keep the Button text as simple and actionable as possible. Refer to the [Button writing guidelines](#Writing) for more detail."
+        defaultCode={`
+  <Button text="Create account" size="lg" color="red" />
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Do not add icons to a Button to refinforce the text."
+        defaultCode={`
+          <Button text="Create new Pinterest account" size="lg" color="red" iconEnd='person-add' />
+`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
 );
 
 card(
-  <Example
-    name="Size"
-    id="size"
-    defaultCode={`<Flex gap={2}>
-  <Button size="sm" text="Small-sized button" />
-  <Button text="Medium-sized button" />
-  <Button size="lg" text="Large-sized button" />
-</Flex>`}
-  />,
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="ARIA attributes"
+      description={`
+When Button text does not provide sufficient context about the Button’s behavior, supply a short, descriptive label for screen-readers using accessibilityLabel. Texts like “Click here“, “Follow“, or “Shop“ can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text with deeper context to replace the Button text, like “Follow Ryan” or “Shop Wedding Invitations”.
```

Comment:
using `accessibilityLabel`  missing format

Reply:
Texts like “Click...

 should go in new line

Reply:
If Button ... 

needs one more line of space

Reply:
Done!